analyzer/src/funTest/kotlin/integration/AbstractIntegrationSpec.kt

```diff
@@ -106,7 +106,8 @@ abstract class AbstractIntegrationSpec : StringSpec() {
 
                 results.size shouldBe files.size
                 results.values.forEach { result ->
-                    result.project.vcs.provider shouldBe pkg.vcs.provider
+                    VersionControlSystem.forProvider(result.project.vcs.provider) shouldBe
+                            VersionControlSystem.forProvider(pkg.vcs.provider.toLowerCase())
```

Comment:
Now you don't need the `.toLowerCase()` here anymore.

Reply:
Done.